doc/tools/pkcs11-tool.1.xml

```diff
@@ -75,8 +75,16 @@
 					<term>
 						<option>--hash-algorithm</option> <replaceable>mechanism</replaceable>
 					</term>
-					<listitem><para>Specify hash algorithm used with
-					RSA-PKCS-PSS signature. Default is SHA-1.</para></listitem>
+					<listitem><para>
+	  Specify hash algorithm used with RSA-PKCS-PSS signature or RSA-OAEP decryption.
+          Allowed values are "SHA-1", "SHA256", "SHA384", "SHA512". Default is SHA-1.
+	  Note that the input to RSA-PKCS-PSS has to be of
+          the size equal to the specified hash algorithm. E.g., for SHA256 the signature input must be
+          exactly 32 bytes long (for mechanisms SHA256-RSA-PKCS-PSS there is no such restriction).
+          For RSA-OAEP, the plaintext input size mLen must be at most
+          keyLen - 2 - 2*hashLen. For example, for RSA 3072-bit key and SHA384, the longest plaintext
+          to encrypt with RSA-OAEP is (with all sizes in bytes): 384 - 2*48 -2 = 286, aka 286 bytes.
+          </para></listitem>
```

Comment:
All the other lines are indented under the opening tag, which makes it look like waste of space on the screen, but makes it possible to orient inside of the file for future editors. I would recommend to indent also this block with 5 tabs.
Possibly split the paragraphs into two separate paragprahs using `<para>` tag should do the job.
And just a nitpick, there should come space in front of the two in term `-2`.

Reply:
Addressed, thanks.